<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="hSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="files">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSplitter" name="vSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPlainTextEdit" name="editor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
       <widget class="QTextEdit" name="log">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
    <addaction name="action_build"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="actionQt"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_B"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_quit"/>
   <addaction name="separator"/>
   <addaction name="action_cut"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_build"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_new">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/folder_new.png</normaloff>:/res/folder_new.png</iconset>
   </property>
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/open.png</normaloff>:/res/open.png</iconset>
   </property>
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/save.png</normaloff>:/res/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/exit.png</normaloff>:/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/undo.png</normaloff>:/res/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/redo.png</normaloff>:/res/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/cut.png</normaloff>:/res/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/copy.png</normaloff>:/res/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/paste.png</normaloff>:/res/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_build">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/build.png</normaloff>:/res/build.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/help.png</normaloff>:/res/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
